### ISSUE TITLE
Fix Request: HttpClientBuilderWithoutPoolSize false positive

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#159]
Example code:
 ```
public static void registerMBean(PoolingHttpClientConnectionManager connMgr) { //NOPMD - suppressed HttpClientBuilderWithoutPoolSize - reason: false positive - not building
        PoolingHttpClientConnectionManagerStatsMXBean poolStats = new PoolingHttpClientConnectionManagerStats(connMgr);
        // ...
    }
```
